compiler/rustc_codegen_cranelift/src/intrinsics/mod.rs

```diff
@@ -497,6 +497,8 @@ pub(crate) fn codegen_intrinsic_call<'tcx>(
         truncf64(flt) -> f64 => trunc,
         roundf32(flt) -> f32 => roundf,
         roundf64(flt) -> f64 => round,
+        roundevenf32(flt) -> f32 => roundevenf,
+        roundevenf64(flt) -> f64 => roundeven,
```

Comment:
If I understand correctly, this will cause calls to the `roundeven` and `roundevenf` libc functions. But those are non-standard, and not available on all libc implementations.

cc @bjorn3

Reply:
Cranelift doesn't have an instruction for this. If this ever becomes a problem, I can request one.